t/youtube.t

```diff
@@ -29,10 +29,10 @@ sub check_video_fetch_url {
     is $code, 200;
 }
 
-# YAPC video 1	
+# YAPC video 1
 check_video_fetch_url('Y1I1KcKvz9Q');
 
-# YAPC video 2 
+# YAPC video 2
```

Comment:
Thanks for fixing the line endings here. Do any of  the examples in this test trigger `_is_new()`? It would be helpful to cover both cases. This test will only be run if `P5_YOUTUBE_NETWORK_TESTS`, so we don't have to worry about the average test case for the end user.

Reply:
`t/data/player_response.html` is trimmed from `curl -o ...` the first YAPC video. Getting the *old* content as a test might be a challenge.